lib/mobility/plugins/active_record/dirty.rb

```diff
@@ -51,9 +51,11 @@ module Dirty
           end
         end
 
-        included_hook do |_, backend_class|
+        included_hook do |klass, backend_class|
           if options[:dirty]
             backend_class.include BackendMethods
+            # I'm not sure why this is now public, but it is ¯\_(ツ)_/¯
+            klass.send(:public, :changes_applied) if ::ActiveRecord::VERSION::STRING >= '7.0'
```

Comment:
This has nothing to do with this PR, but specs on AR 7.0 are failing unless this is fixed, so sneaking it in.

Reply:
Ok here's what changed this: https://github.com/rails/rails/pull/41860

Reply:
Ok this is a Rails problem, I'll wait for it to be fixed and rebase. We shouldn't need this patch.